server/requeststojointeamhandler.go

```diff
@@ -0,0 +1,93 @@
+package server
```

Comment:
i think the filenename should mirror the handler name : `*list*requeststojointeamhandler.go`